stories/Icons.story.tsx

```diff
@@ -0,0 +1,325 @@
+import * as React from 'react';
+
+import {
+    description,
+    importExample,
+    header,
+    divider,
+    tab,
+    tabs,
+    title,
+    code as baseCode
+} from 'wix-storybook-utils/Sections';
+
+
+import { allComponents } from './utils/allComponents';
+
+const code = config =>
+    baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+    category: 'Icons',
+    storyName: 'Icons',
+    sections: [
+        header(),
+        tabs([
+            tab({
+                title: 'Icon Inventory',
+                sections: [
+                    description({
+                        title: 'Importing the icons',
+                        text: [
+                            'We are using the ',
+                            <a href="https://www.npmjs.com/package/@svgr/webpack" >
+                                svgr plugin
+                            </a>,
+                            ' for svg imports which allow you to import the svg file directly. ',
+                            'If you have ',
+                            <a href="https://webpack.js.org/guides/tree-shaking/" >
+                                Tree Shaking
+                            </a>,
+                            ' in your application use the first option otherwise use the second option.'
+                        ],
+                    }),
+
+
+                    importExample(
+                        "import { Icons } from 'wix-ui-tpa';\n" +
+                        "<Icons.Calendar />",
+                    ),
```

Comment:
Just to make a bit clearer, write something like this?
```suggestion
                    importExample(
                        `import { Icons } from 'wix-ui-tpa';
                        ...
                        render() {
                          ...
                          return (
                            ...  
                            <Icons.Calendar />,
                            ...
                          );
                        }`
                    ),
```

Or something like that?